### PR TITLE
add title attributes to navbar links

### PR DESCRIPTION
**Pull request by Baalwaan**

### CODE REVIEW

File: src/components/common/header/index.js [hunk 1]
```diff
@@ -10,47 +17,47 @@ import { ReactComponent as LogoYellow } from '../../../assets/logo-yellow.svg';
 import { ReactComponent as Burger } from '../../../assets/burger-icon.svg';
 
 const Wrapper = styled.section`
-    // position: fixed;
```
Comment (tloth) [inline src/components/common/header/index.js]: please delete or uncomment

File: src/components/common/header/index.js [hunk 2]
```diff
@@ -10,47 +17,47 @@ import { ReactComponent as LogoYellow } from '../../../assets/logo-yellow.svg';
...
-    top: 0;
-    display: flex;
-    background-color: ${colours.black};
-    color: ${colours.white};
-    /* width: 100%; */
-    padding: ${spacing.tiny};
-    justify-content: space-between;
+  // position: fixed;
```
Comment (tloth) [inline src/components/common/header/index.js]: please delete or "uncomment"

File: src/components/common/header/index.js [hunk 3]
```diff
@@ -10,47 +17,47 @@ import { ReactComponent as LogoYellow } from '../../../assets/logo-yellow.svg';
...
+  top: 0;
+  display: flex;
+  background-color: ${colours.black};
+  color: ${colours.white};
+  /* width: 100%; */
```
Comment (tloth) [inline src/components/common/header/index.js]: please delete or "uncomment"

File: src/components/common/header/index.js (hunk 2 above)
Comment (Baalwaan) [reply src/components/common/header/index.js]: 🙌 Nice one! all done

**Pull request merged**